### PR TITLE
webtesting: added to project

### PR DESCRIPTION
**Pull request by gjstoychev**
Added Task1 and Task2 from Web Testin Subjects

### CODE REVIEW

File: webtesting/Task1/assets/app.bundle.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (mgenov) [inline webtesting/Task1/assets/app.bundle.js]: .

File: webtesting/Task1/src/calc.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+var CalcOperations;
+
+var number = "",
+    total = 0,
+    regexp = /[0-9]/,
+    hasDot = false;
+
+CalcOperations = {
```
Comment (mgenov) [inline webtesting/Task1/src/calc.js]: Naming could be improved with something like: 

```javascript
Calculator.onEnterSymbol
Calculator.onEnterDot(num)
Calculator.onTotal() 
```


Also the current state could be kept in the calculator and the usage will be simplified to something like: 

```javascript
let calculator = Calculator()
calculator.onEnterSymbol('1')
let total = calculator.onTotal()
assert(total).equalTo('1') 
```

File: webtesting/Task1/assets/app.bundle.js (hunk 1 above)
Comment (gjstoychev) [inline webtesting/Task1/assets/app.bundle.js]: I know it is a good practice to git ignore bundled dist folders, but I commit it on purpose to let you see the new created .js without building it.

File: webtesting/Task1/src/calc.js (hunk 2 above)
Comment (gjstoychev) [inline webtesting/Task1/src/calc.js]: Naming fixed.
Functions simplified.

File: webtesting/Task1/assets/app.bundle.js (hunk 1 above)
Comment (mgenov) [inline webtesting/Task1/assets/app.bundle.js]: Ok, so you can remove it in following PR :)

**Comment by mgenov** [comment]
Please squash your commits.

**Comment by gjstoychev** [comment]
Squashed.

**Pull request merged**